[Bug] Installation fails when building from source

### Describe the bug

After
```
make
sudo make install
```
 I got:
 ```
-- Up-to-date: /usr/local/include/zenoh-pico/session/subscription.h
-- Up-to-date: /usr/local/include/zenoh-pico/session/publication.h
CMake Error at build/cmake_install.cmake:65 (file):
  file cannot create directory: /cmake/zenohpico.  Maybe need administrative
  privileges.
```

### To reproduce

Follow the instructions in the readme for building from source

### System info

OS: macOS 13.4.1 22F82 arm64

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,6 +16,7 @@ cmake_minimum_required(VERSION 3.13)
 project(zenohpico VERSION 0.10.0.0 LANGUAGES C)
 
 include(CMakePackageConfigHelpers)
+include(GNUInstallDirs)
 
 option(BUILD_SHARED_LIBS "Build shared libraries if ON, otherwise build static libraries" ON)
 option(ZENOH_DEBUG "Use this to set the ZENOH_DEBUG variable." 0)

